unity-renderer/Assets/Scripts/Editor/BuildCommand.cs

```diff
@@ -141,12 +141,10 @@ static string getEnv(string key, bool secret = false, bool verbose = true)
 
     static void PerformBuild()
     {
-        Console.WriteLine(":: Building addressables");
+           Console.WriteLine(":: Building addressables");
 
         AddressableAssetSettings.CleanPlayerContent(
             AddressableAssetSettingsDefaultObject.Settings.ActivePlayerDataBuilder);
-        AddressableAssetSettings.BuildPlayerContent();
```

Comment:
🤔 this is correct?

Reply:
Im using it as assets. I dont want them to get build to confirm that the safe works.